src/components/ContributorAllocations.js

```diff
@@ -88,7 +89,16 @@ const ContributorAllocations = (props) => {
                 </Grid>
                 <Grid item xs={12}>
                     <Grid container spacing={5}>
-                        {renderAllocations({ allocations: contributorAllocations.allocations })}
+                        {
+                            contributorAllocations.allocations.length
```

Comment:
This logic is confusing because it's not clear whether you are trying to render if there is more than 1 allocation in the array or if you are simply checking whether the value exists or is defined or is an array which would be better written more explicitly with `Array.isArray(contributorAllocations.allocations)`

Please resolve

Reply:
@otech47 I change this and since what I'm trying to verify is if the array contains elements now I use the lodash function `isEmpty` that does the same but it's more readable